lib/moment-range.js

```diff
@@ -353,6 +353,27 @@ moment.range = function(start, end) {
 moment.range.constructor = DateRange;
 
 /**
+ * Build a date range between a specified interval and now (or a specified date).
+ *
+ * @param {String} interval The type of interval
+ * @param {Number} [count=1] The number of intervals
+ * @param {(Moment|Date} [date=moment()] The date to use
+ *
+ * @return {!DateRange}
+ */
+moment.range.fromInterval = function(interval, count = 1, date = moment()) {
+  if (!moment.isMoment(date)) date = moment(date);
+
+  let date2 = date.clone().add(count, interval);
+  let dates = [];
+
+  dates.push(moment.min(date, date2));
+  dates.push(moment.max(date, date2));
+
+  return new DateRange(dates);
```

Comment:
I think you could just `return new DateRange([date, date2]);` and remove the several lines above that are building up the array.